bootstrap-templates/default/secrets.yaml

```diff
@@ -0,0 +1,31 @@
+transit-key-name: sanctum-default
```

Comment:
I'd consider templating this to something like `{{ .Params.name }}-transit`, but I don't feel strongly.

Reply:
Noted. I'm going to merge this as-is because I also don't have strong feelings on using separate transit keys per project given our current levels of access control.